kinetic/ros-core/Dockerfile

```diff
@@ -1,3 +1,29 @@
-FROM alpineros/alpine-ros:kinetic-bare
```

Comment:
Just a question, is there any reason why you don't use `alpineros/alpine-ros:kinetic-bare` any more?

Reply:
On docker hub, such kind of dependency is not automatically solved. Manual build trigger is required to build all images on the previous configuration.

Reply:
Got it. Thanks.